mlflow/store/tracking/abstract_store.py

```diff
@@ -36,6 +36,34 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY, max_results=None, pag
         """
         pass
 
+    @abstractmethod
+    def search_experiments(
+        self,
+        view_type=ViewType.ACTIVE_ONLY,
+        max_results=None,
+        filter_string=None,
+        order_by=None,
+        page_token=None,
+    ):
+        """
+        Search for experiments that match the specified search criteria.
+
+        :param view_type: Qualify requested type of experiments.
+        :param max_results: If passed, specifies the maximum number of experiments desired. If not
+                            passed, all experiments will be returned for the File and SQL backends.
+                            For the REST backend, the server will pick a maximum number of results
+                            to return.
+        :param filter_string: Filter query string, defaults to searching for all experiments.
```

Comment:
Let's describe which fields (attribute.*, tag.*) can be used in filter and which comparators are allowed.

Reply:
Done